sycl/include/CL/sycl/detail/property_helper.hpp

```diff
@@ -47,7 +47,8 @@ enum PropWithDataKind {
   ImageUseMutex = 2,
   ImageContextBound = 3,
   BufferMemChannel = 4,
-  PropWithDataKindSize = 5
+  AccPropBufferLocation = 5,
+  PropWithDataKindSize = 6,
```

Comment:
Is there any reason why new enum is not added at the end?

Reply:
Feel free to ignore, I found the reason: there is a comparison somewhere where all other enum of the same type cannot exceed this last enum.

https://github.com/intel/llvm/blob/54ede407edeb93b7e9334d1e725f48bf981b2965/sycl/include/CL/sycl/detail/property_list_base.hpp#L94

Reply:
Is this an ABI breaking change given that PropWithDataKindSize  changed from 5 -> 6?

Reply:
It is not an ABI breaking change, because it is not a property and it is used only in `property_list_base.hpp`